lib/api/v1/queries/keywords.go

```diff
@@ -36,3 +36,15 @@ func (q *KeywordsQuery) Where(queryParams KeywordsQueryParams) ([]models.Keyword
 
 	return keywords, nil
 }
+
+func (q *KeywordsQuery) Find(id string) (*models.Keyword, error) {
+	db := database.GetDB()
+
+	var keyword models.Keyword
+	err := db.First(&keyword, "id = ?", id).Error
```

Comment:
```suggestion
	err := db.First(&keyword, id).Error
```

Gorm has a nice documentation 😇 
https://gorm.io/docs/query.html#Retrieving-objects-with-primary-key

Reply:
@malparty because it's a string so we have to follow this suggestion from Gorm
```
If the primary key is a string (for example, like a uuid), the query will be written as follows:

db.First(&user, "id = ?", "1b74413f-f3b8-409f-ac47-e8c062e3472a")
// SELECT * FROM users WHERE id = "1b74413f-f3b8-409f-ac47-e8c062e3472a"
```